src/dos/drive_iso.cpp

```diff
@@ -342,6 +342,15 @@ bool isoDrive::GetFileAttr(char *name, Bit16u *attr) {
 	return success;
 }
 
+bool isoDrive::SetFileAttr(const char * name, uint16_t /*attr*/) {
```

Comment:
`uint16_t /*attr*/)` -> `[[maybe_unused]] uint16_t attr)`

Good news is that `[[maybe_unused]]` is part of the C++17 language and is a way of tell the compiler and reader that a variable is deliberately not used (or maybe not used; in the case of debug/optimized scenarios and such).

Reply:
Done.